{{ cookiecutter.__package_name_kebab_case }}/.gitlab-ci.yml

```diff
@@ -67,16 +68,14 @@ Compute Image Hash:
         - pyproject.toml
 
 # Build CI Docker image.
-Build CI:
+Build CI image:
   extends:
     - .docker
   stage: build
   variables:
     DOCKER_IMAGE: $CI_REGISTRY_IMAGE/ci
+    DOCKER_IMAGE_SHA: $CI_IMAGE_SHA
     DOCKER_TARGET: ci
-  artifacts:
-    reports:
-      dotenv: .env
```

Comment:
These lines are not needed because the effect is already provided by lines 11-13 in the previous stage [1].

[1] https://gitlab.com/gitlab-org/gitlab/-/issues/246777#note_409025727